src/main/kotlin/net/ccbluex/liquidbounce/render/RenderShortcuts.kt

```diff
@@ -289,7 +302,7 @@ fun BufferBuilder.coloredTriangle(matrix: Matrix4f, p1: Vec3d, p2: Vec3d, p3: Ve
  * @param box The bounding box of the side.
  * @param side The direction of the side.
  */
-fun RenderEnvironment.drawSideBox(box: Box, side: Direction) {
+fun RenderEnvironment.drawSideBox(box: Box, side: Direction, drawMode: DrawMode = DrawMode.QUADS) {
```

Comment:
There is only one mode other than `QUADS` that viable for this function ("DEBUG_LINE_STRIP"). When a new user sees this function, it might be hard to understand what behaviour to expect and what features this function has. So I'd recommend changing this `drawMode` parameter to `onlyOutlines: Boolean` or something so it immediately becomes obvious what the line `drawSideBox(..., onlyOutlines=true)` means ;)